pkgs/tools/video/yamdi/default.nix

```diff
@@ -0,0 +1,31 @@
+{
+  stdenv,
+  fetchurl,
+}:
+
+stdenv.mkDerivation rec {
+  name = "yamdi-${version}";
+  version = "1.9";
+
+  # Source repo is also available here:
+  # https://github.com/ioppermann/yamdi
+  src = fetchurl {
+    url = "https://downloads.sourceforge.net/project/yamdi/yamdi/${version}/yamdi-${version}.tar.gz";
```

Comment:
I'am sorry I forgot this:
You can replace this with `url = "mirror://sourceforge/yamdi/yamdi-${version}.tar.gz";`
We do have some special mirrors in nixpkgs. This helps a lot with changing download URLs/procedures/redirects or unavailable servers. Especially sourceforge can be a pain with that.
Btw for your future derivations: There also is a `fetchgit` and `fetchFromGitHub` to directly fetch source repos. `fetchgit` does checkout submodules as well. Do never use fetchurl with github zip archives, as github is known for recompressing archives rendering checksums invalid.